tests/wp-unit/include/Core/Posts/Modules/DeletePostsByStatusModuleTest.php

```diff
@@ -398,4 +398,99 @@ public function test_that_draft_posts_can_be_deleted_in_batches() {
 		$trash_posts = $this->get_posts_by_status( 'trash' );
 		$this->assertEquals( 50, count( $trash_posts ) );
 	}
+
+	/**
+	 * Helper Function.
+	 * Create posts from a custom post status
+	 */
+	public function create_posts_by_custom_status( $post_status = "custom", $count = 10 ) {
+		register_post_status( $post_status );
```

Comment:
Move the call to `register_post_status` outside of the helper function and combine `create_posts_by_custom_status` and `create_post_by_scustom_tatus` into a single function.

You also need to fix the typo in the function name.